applications/community-health-check.md

```diff
@@ -0,0 +1,281 @@
+
+At this stage, we do not provide sentiment or keyword analysis at the community or interaction level. We start with a neutral perspective on interaction (two people interacting with each other) and do not consider finer nuances (is the interaction positive or negative).
+
+
+- An overview of the technology stack to be used
+Our dashboard is built using a combination of Nextjs (Typescript). Also, we use Python for analytics & ExpressJs for the backend. Data is stored in a MongoDB.
+
+- Documentation of core components, protocols, architecture, etc. to be deployed
+Installation instructions for the first version of our community health checks: 
+- data extraction bot: https://www.notion.so/netnigma/Community-Health-bot-Tutorial-94a4b089681f41aeb028e1bffd25aaad
+- DIY health checks: https://www.notion.so/netnigma/Getting-started-with-the-Health-Pulse-Onboarding-guide-8545d383ec5d4d70aeb4b2aae22d97d7
+
+
+- PoC/MVP or other relevant prior work or research on the topic
+We published our research framework ([Long version](https://rndao.mirror.xyz/F-SMj6p_jdYvrMMkR1d9Hd6YbEg39qItTKfjo-zkgqM); [summary](https://rndao.mirror.xyz/fsCGRlYR5rieDCVPnAnSAGL98wNWJyFbf1T_A2r-_6Y))
+
+We did a health check for MetaGame, who tweeted it [here](https://twitter.com/MetaFam/status/1603818967528718336).
+
+We are running the community health check on our small team. Our low fidelity [dashboard](https://www.figma.com/proto/jflSBwHW3b5KgohsaYExuN/TogetherCrew?scaling=scale-down-width&page-id=0%3A1&starting-point-node-id=1%3A49&show-proto-sidebar=1&node-id=1%3A49&hide-ui=1) is available.
+
+
+
+### Ecosystem Fit
+
+- Where and how does your project fit into the ecosystem?
+Our project gives the ecosystem information about its health by measuring the interactions (conversations, relationships) between contributors. We consider this a "soft performance indicator", because it doesn't measure objective aspects (e.g., performance, dapps launched, bugs fixed) but centers on the humans of the ecosystem. 
+
+- Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+The community health check is for community managers and moderators to support them in their work by providing community metrics.
+
+- What need(s) does your project meet?
+Our project provides moderators, leaders, and contributors with a bird's eye view of what is happening inside the ecosystem. 
+
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+Our current approach rests on off-chain data. Other community analytics dashboard exists, fetching data from various off-chain data. However, they focus on posting behavior and not on the  interaction between members. Orbit.love reach metric, conceptually, comes close to our metrics, but technically also rests on individual posting behavior and not interaction data. 
```

Comment:
> but technically also rests on individual posting behavior and not interaction data

Could you further elaborate on the difference between _individual posting behavior_ and _interaction data_? Given a network graph where the nodes represent the SN's participants and the edges between them their relationships (e.g. likes, comments, follows, etc.), does it mean that the further only focuses on the nodes while your solution would additionally take the edges into account?

Reply:
Yes you said it well. Individual posting behavior focuses only on the number of activities of a person, but not how these activities impact others. Edges are our interaction data (replies, mentions, emoji reactions)

Reply:
Thanks. If that's the case, could you please incorporate this information into the doc?